src/drivers/heater/heater.h

```diff
@@ -163,9 +164,32 @@ class Heater : public ModuleBase<Heater>, public ModuleParams, public px4::Sched
 	 */
 	void update_params(const bool force = false);
 
+	/**
+	 * @brief Enables / configures the heater (either by GPIO or PX4IO)
+	 */
+	void heater_enable(void);
+
+	/**
+	 * @brief Disnables the heater (either by GPIO or PX4IO)
+	 */
+	void heater_disable(void);
+
+	/**
+	 * @brief Turns the heater on (either by GPIO or PX4IO)
+	 */
+	void heater_on(void);
+
+	/**
+	 * @brief Turns the heater off (either by GPIO or PX4IO)
+	 */
+	void heater_off(void);
+
 	/** Work queue struct for the RTOS scheduler. */
 	static struct work_s _work;
 
+	/** File descriptor for PX4IO for heater ioctl's */
+	int _io_fd;
```

Comment:
```suggestion
#if defined(HEATER_PX4IO)
	int _io_fd{-1};
#endif // HEATER_PX4IO
```

Reply:
HEATER_PX4IO will be defined in heater.cpp which includes heater.h before it could be defined.